projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -89,6 +89,14 @@ public function reregister_menu_items() {
 	 * @return string
 	 */
 	public function get_preferred_view( $screen, $fallback_global_preference = true ) {
+		/**
+		 * When Jetpack SSO is disabled, we need to force Calypso because it might create confusion to be redirected to WP-Admin.
+		 * Furthermore, because we don't display the quick switcher, users having an WP-Admin interface by default won't be able to go back to the Calyso version.
+		 */
+		if ( ! \Jetpack::is_module_active( 'sso' ) ) {
+			return self::DEFAULT_VIEW;
+		}
+
```

Comment:
This needs to be placed below the exception for Plugins, Export and Customize, since these pages are not supported in Calypso for Atomic sites, and they should always link to WP Admin regardless of the SSO module.

Reply:
Good catch! I'll update the logic right now, thanks! :)